p2p/libp2p/mutexHolder.go

```diff
@@ -0,0 +1,50 @@
+package libp2p
+
+import (
+	"sync"
+
+	"github.com/ElrondNetwork/elrond-go/storage/lrucache"
+)
+
+// MutexHolder holds a cache of mutexes: pairs of (key, *sync.Mutex)
+type MutexHolder struct {
+	//generalMutex is used to assure access to the already concurrent safe lrucache is being serialized
```

Comment:
//generalMutex is used to serialize the access to the already concurrent safe lrucache

Reply:
changed